src/ai/backend/manager/models/container_registry.py

```diff
@@ -154,19 +176,20 @@ async def get_known_container_registries(
         return result
 
     @classmethod
-    def from_dataclass(cls, image_data: ContainerRegistryData) -> Self:
-        return cls(
-            id=image_data.id,
-            url=image_data.url,
-            registry_name=image_data.registry_name,
-            type=image_data.type,
-            project=image_data.project,
-            username=image_data.username,
-            password=image_data.password,
-            ssl_verify=image_data.ssl_verify,
-            is_global=image_data.is_global,
-            extra=image_data.extra,
+    def from_dataclass(cls, data: ContainerRegistryData) -> Self:
+        instance = cls(
+            url=data.url,
+            registry_name=data.registry_name,
+            type=data.type,
+            project=data.project,
+            username=data.username,
+            password=data.password,
+            ssl_verify=data.ssl_verify,
+            is_global=data.is_global,
+            extra=data.extra,
         )
+        instance.id = data.id
+        return instance
```

Comment:
Why don't you put the ID value at creation time?

Reply:
Since the ID is injected from the database, the ID value passed through the constructor wouldn't be passed to the database, so I intentionally excluded it from the constructor.